### PR TITLE
Port previous work into existing structure

### PR DESCRIPTION
**Pull request by bethqiang**
Previous work includes A-Frame and socket code. Also, delete React.

### CODE REVIEW

File: browser/index.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+socket.on('getOthersCallback', users => {
+  console.log('Checking to see if anyone is here');
+  for (let i = 0; i < users.length; i++) {
+    const avatar = document.createElement('a-box');
+    scene.appendChild(avatar);
+    avatar.setAttribute('id', users[i].id);
+    avatar.setAttribute('color', users[i].color);
+    avatar.setAttribute('position', `${users[i].x} ${users[i].y} ${users[i].z}`);
+    avatar.setAttribute('rotation', `${users[i].xrot} ${users[i].yrot} ${users[i].zrot}`);
+  }
```
Comment (JoeyDarbyshire) [inline browser/index.js]: can this be changed to a users.map instead of a for loop?

File: browser/index.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+socket.on('newUser', user => {
+  console.log('Someone else has joined');
+  const avatar = document.createElement('a-box');
+  scene.appendChild(avatar);
+  avatar.setAttribute('id', user.id);
+  avatar.setAttribute('color', user.color);
+  avatar.setAttribute('position', `${user.x} ${user.y} ${user.z}`);
+  avatar.setAttribute('rotation', `${user.xrot} ${user.yrot} ${user.zrot}`);
+});
+
```
Comment (JoeyDarbyshire) [inline browser/index.js:54]: We should modulize this since the same code is being used above

File: browser/index.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+socket.on('usersUpdated', users => {
+  console.log('Updating position for all users');
+  for (let i = 0; i < users.length; i++) {
+    const otherAvatar = document.querySelector(`#${users[i].id}`);
+    otherAvatar.setAttribute('position', `${users[i].x} ${users[i].y} ${users[i].z}`);
+    otherAvatar.setAttribute('rotation', `${users[i].xrot} ${users[i].yrot} ${users[i].zrot}`);
+  }
+});
```
Comment (JoeyDarbyshire) [inline browser/index.js:69]: Again, maybe a users.map?

**Review by JoeyDarbyshire — APPROVED** [review]

File: browser/index.js (hunk 1 above)
Comment (bethqiang) [reply browser/index.js]: Changed to forEach, per our conversations.

File: browser/index.js (hunk 3 above)
Comment (bethqiang) [reply browser/index.js:69]: Changed to forEach.

**Review by JoeyDarbyshire — APPROVED** [review]

**Pull request merged**